### ISSUE TITLE
MLforecast does not work with with PyArrow dates

### ISSUE THREAD

**Issue by nprihodko** [Nixtla/mlforecast#305]
### What happened + What you expected to happen

When the date column in the data frame supplied to MLForecast is PyArrow data type, it does not work. MLForecast throws error:

```
ValueError: The time column ('ds') should have either timestamps or integers, got 'object'.
```

I would expect either
- MLForecast working with PyArrow dtypes,
- or explicit disclaimer that only NumPy dtypes are supported.

### Versions / Dependencies

- Python 3.10.13
- MLForecast 0.11.6
- MacOS 13.4.1

### Reproduction script

```python
import lightgbm as lgb
from mlforecast import MLForecast
from mlforecast.utils import generate_daily_series

# Generate some data
df = generate_daily_series(
    n_series=20,
    max_length=100,
    n_static_features=0,
    static_as_categorical=False,
    with_trend=True,
)

# Convert to PyArrow dtypes
df = df.astype(
    {
        "ds": "timestamp[ns][pyarrow]",
        "unique_id": "string[pyarrow]",
        "y": "float64[pyarrow]",
    }
)
print(df.dtypes)

# Create MLForecast object 
models = [lgb.LGBMRegressor(verbosity=-1)]
fcst = MLForecast(models=models, freq="D", lags=[7, 14])

# Fit
fcst.fit(df)
```

### Issue Severity

Medium: It is a significant difficulty but I can work around it.

**Comment by jmoralez** [comment]
Hey @nprihodko, thanks for using mlforecast and for the detailed report. This should be fixed in utilsforecast 0.0.26. Can you please upgrade to that version and verify (currently only in PyPI)?

**Comment by nprihodko** [comment]
@jmoralez, this seems to fix the issue. Thank you!
